src/Resources/views/Core/search.html.twig

```diff
@@ -26,15 +26,18 @@ file that was distributed with this source code.
                 {% for admin in group.items %}
                     {% set count = count + 1 %}
                     {% if admin.hasRoute('create') and admin.hasAccess('create') or admin.hasRoute('list') and admin.hasAccess('list') %}
+                        {# NEXT_MAJOR: Remove the admin_code param. #}
                         {{ sonata_block_render({
                             'type': 'sonata.admin.block.search_result'
                         }, {
                             'query': query,
                             'admin_code': admin.code,
+                            '_sonata_admin': admin.baseCodeRoute,
```

Comment:
This is a block, I think it doesn't apply or the block should be modified.

Reply:
Indeed, removed.